src/main/resources/reference.conf

```diff
@@ -1,106 +1,246 @@
 akka.http.server.idle-timeout = 180 s
 akka.http.server.request-timeout = 60 s
 
-resourceTypes =
-  {
-    workspace = {
-      actionPatterns = ["delete", "read_policies", "share_policy::owner", "share_policy::writer", "share_policy::reader", "read_policy::owner", "read_policy::project-owner"]
-      ownerRoleName = "owner"
-      roles = {
-        owner = {
-          roleActions = ["delete", "read_policies", "share_policy::owner", "share_policy::writer", "share_policy::reader", "read_policy::project-owner"]
-        },
-        writer = {
-          roleActions = ["read_policy::owner", "read_policy::project-owner"]
-        },
-        reader = {
-          roleActions = ["read_policy::owner", "read_policy::project-owner"]
-        }
-      }
-      reuseIds = false
-    },
-    managed-group = {
-      actionPatterns = ["delete", "read_policies", "share_policy::admin", "share_policy::member", "read_policy::admin", "read_policy::member"]
-      ownerRoleName = "admin"
-      roles = {
-        admin = {
-          roleActions = ["delete", "read_policies", "share_policy::admin", "share_policy::member", "read_policy::admin", "read_policy::member"]
-        },
-        member = {
-          roleActions = []
-        }
-      }
-      reuseIds = true
-    },
-    workflow-collection = {
-      actionPatterns = ["delete", "add", "view", "abort", "update", "alter_policies", "read_policies"]
-      ownerRoleName = "owner"
-      roles = {
-        owner = {
-          roleActions = ["delete", "add", "view", "abort", "update", "alter_policies", "read_policies"]
-        },
-        reader = {
-          roleActions = ["view"]
-        },
-        writer = {
-          roleActions = ["view", "add", "delete", "abort", "update"]
-        }
-      }
-      reuseIds = false
-    },
-    caas = {
-      actionPatterns = ["get_whitelist", "alter_policies", "read_policies"]
-      ownerRoleName = "owner"
-      roles = {
-       owner = {
+resourceTypes = {
+  workspace = {
+    actionPatterns = {
+      delete = {
+        description = "Delete this workspace"
+      }
+      read_policies = {
+        description = "View all policies and policy details for this workspace"
+      }
+      "share_policy::owner" = {
+        description = "Add/remove members to/from the owner policy for this workspace"
```

Comment:
how about "may change the membership of the owner policy for this workspace"

I find "add/remove" and "to/from" too distracting